src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1747,6 +1748,13 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
                             OutputProcessor::SOVTimeStepType::System,
                             OutputProcessor::SOVStoreType::Average,
                             this->name);
+        SetupOutputVariable(state,
+                            "Heat Pump Source Side Specific Heat",
+                            OutputProcessor::Unit::J_kgK,
+                            this->sourceSideCp,
+                            OutputProcessor::SOVTimeStepType::System,
+                            OutputProcessor::SOVStoreType::Average,
+                            this->name);
```

Comment:
Register the output variable.

Reply:
I noticed that this report is only for the electric EIR HP. Is this intended to be used for fuel-fired also? And why are the source side reports commented out for the fuel-fired model (note Heat Pump Source Side Outlet Temperature is wrong there, should be Fuel-fired)? see EIRFuelFiredHeatPump::oneTimeInit.

Reply:
> I noticed that this report is only for the electric EIR HP. Is this intended to be used for fuel-fired also? And why are the source side reports commented out for the fuel-fired model (note Heat Pump Source Side Outlet Temperature is wrong there, should be Fuel-fired)? see EIRFuelFiredHeatPump::oneTimeInit.

My guess is that the commented output variables are not calculated because the fuel-fired heat pump is air-source and likely just uses the air temp for curve variables without any energy balance on the source side? Is that correct @jcyuan2020 ?  It does seem that "Fuel-fired Absorption Heat Pump Source Side Inlet Temperature" could be reported?

Reply:
@mjwitte @rraustad That's right. Fuel-fired heat pump was developed for air-source only (probably due to the original client's request). It is not a 100% fuel-energy resource version of the original electric heat pump it is based on--there are features trimmed down (e.g. air/water source vs. air-source) and added in (e.g. constant flow vs. variable flows/flow controls). @mjwitte These might become some new development topics to catch up the have features for these two model(?).